notebook/static/edit/js/editor.js

```diff
@@ -76,12 +76,20 @@ function(
     // default CodeMirror options
     Editor.default_codemirror_options = {
         extraKeys: {
-            "Tab" :  "indentMore",
+            "Cmd-Right": "goLineRight",
+            "End": "goLineRight",
+            "Cmd-Left": "goLineLeft",
+            "Tab": "indentMore",
+            "Shift-Tab" : "indentLess",
+            // "Cmd-Alt-[" : "indentAuto",
+            // "Ctrl-Alt-[" : "indentAuto",
```

Comment:
Should these be in or out? If we don't want them, let's remove them completely rather than commenting them.

Reply:
I'll remove...

Reply:
Looks like you've removed them from cell.js, but not here ;-)